doc/src/build/devnet.md

```diff
@@ -77,14 +80,26 @@ Secret Recovery Phrase : [cap wheat many line human lazy few solid bored proud s
 
 ### Connect to a custom RPC endpoint
 
-If you previously used `sui genesis` with the force option (`-f` or `--force`), your client.yaml file already includes two RPC endpoints: localnet `http://0.0.0.0:9000` and devnet `https://fullnode.devnet.sui.io:443`). You can view the defined environments with the `sui client envs` command, and switch between them with the `sui client switch` command.
+If you previously used `sui genesis` with the force option (`-f` or `--force`), your client.yaml file already includes two RPC endpoints: `localnet` at `http://0.0.0.0:9000` and `devnet` at `https://fullnode.devnet.sui.io:443`. You can view the defined environments with the `sui client envs` command, and switch between them with the `sui client switch` command.
+
+If you previously installed a Sui client that connected to a Sui network, or created a local network, you can modify your existing `client.yaml` to change the configured RPC endpoint.
```

Comment:
you can use the `sui client new-env` to create a new one, and then use `sui client switch --env <ALIAS>` to set that as the active network